File: app/src/main/java/com/github/sewerina/meter_readings/database/AppDatabase.java
package com.github.sewerina.meter_readings.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {ReadingEntity.class, HomeEntity.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {

    public abstract AppDao readingDao();

}

File: app/src/main/java/com/github/sewerina/meter_readings/database/AppDatabase.kt
package com.github.sewerina.meter_readings.database

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

@Database(entities = [ReadingEntity::class, HomeEntity::class], version = 1, exportSchema = false)
@TypeConverters(DateConverter::class)
abstract class AppDatabase : RoomDatabase() {
    abstract fun readingDao(): AppDao
}